### DISCUSSION TITLE
Does the GetPelican.com Use a Pelican Theme or Not?

### DISCUSSION THREAD

**Discussion by iamraf-ca** [getpelican/pelican#3410] [Q&A]
Hello,

I was wondering if the getpelican.com website uses a Pelican theme, and if so, could you let me know which one it is?

Regards,
Raf

**Comment by bittner** [comment]
According to getpelican/pelican-blog#16 and [news from Sept 15, 2022](https://getpelican.com/blog/new-pelican-site-docs-theme-discussions/) the new Pelican website uses a custom theme called "[getpelican](https://github.com/getpelican/pelican-blog/tree/e7e53d8d14a796c5b2b4a70fae25df49353d1743/themes/getpelican)".

## Disclaimer

Note that this is largely a guess, because the [pelican-blog](https://github.com/getpelican/pelican-blog) repository has its latest commit from 3 years ago, while the [latest news](https://getpelican.com/blog/) are from January this year. It appears that the website source code is not public anymore. @justinmayer can you clarify?

**Reply by justinmayer** [reply]
I have had a lot on my plate recently, but one of these days I will get around to pushing the latest news article `.rst` files to the repository.
